src/utils/utf8.c

```diff
@@ -70,6 +80,20 @@ utf8_ctx_create(const char *locale)
 		return NULL;
 	}
 
+	ec = U_ZERO_ERROR;
+	UChar *translit_id;
+	
+	size_t n;
+	n = (strlen(RULE)+1);
+	translit_id = malloc(n * sizeof(UChar));
+	u_strFromUTF8(translit_id, n, NULL, RULE, -1, &ec);
+	ctx->transliterator = utrans_openU(translit_id, -1, UTRANS_FORWARD, NULL, 0, NULL, &ec);
```

Comment:
Why don't you pass `NFKD_RULE` directly instead of making a copy into `translit_id`?

Reply:
@rmind dunno how to do that, this copy pasta..

Reply:
Oh, it's `utrans_openU()`. I think you can use `utrans_open()`, no? See [here](https://unicode-org.github.io/icu-docs/apidoc/released/icu4c/utrans_8h.html#afaa9d751e1d24617ba1071d8d4c887a3), then the first argument is a plain string.

Reply:
**_utrans_open_** is deprecated so I went for **_utrans_openU_** instead.

Reply:
Ugh.. that just seems such an unnecesary complication to require UTF-16 strings. Well.. okay, we have to keep it painful then.